pkg/rehearse/metrics.go

```diff
@@ -84,8 +86,14 @@ func (m *Metrics) RecordChangedPresubmits(presubmits config.Presubmits) {
 	}
 }
 
-func (m *Metrics) RecordOpportunity(toRehearse config.Presubmits, reason string) {
-	for _, jobs := range toRehearse {
+func (m *Metrics) RecordChangedPeriodics(periodics []prowconfig.Periodic) {
+	for _, job := range periodics {
+		m.ChangedPeriodics = append(m.ChangedPeriodics, job.Name)
+	}
+}
+
+func (m *Metrics) RecordOpportunity(presubmits config.Presubmits, periodics []prowconfig.Periodic, reason string) {
```

Comment:
This is a smell -- if you only ever call it with presubmits or periodics lets just have two methods

Reply:
addressed